Task 3/system/Coding/src/app/PositionControl.h

```diff
@@ -25,30 +25,26 @@
 
 /* PROTOTYPES *************************************************************************************/
 /** 
- * @brief starts timer1 to measure the Time in SearchTrackState
+ * @brief updates the sensorvalues, must be called before any of the other functions are beein called
 */
-extern void SearchTrackState_enterStartTimer1(void);
+extern void PositionControl_UpdateSensorValues(void);
```

Comment:
Function name has to start with lowerCase.
```suggestion
extern void PositionControl_updateSensorValues(void);
```